### PR TITLE
[netbsd] core.stdc.fenv: Fix error undefined uint32_t

### PR DESCRIPTION
**Pull request by ibuclaw**

### CODE REVIEW

File: src/core/sys/posix/signal.d [hunk 1]
```diff
@@ -1136,55 +1136,57 @@ else version (NetBSD)
...
+                int   _trap;
+                int   _trap2;
+                int   _trap3;
+            } fault_t fault;
+            struct poll_t
+            {
+                c_long _band;
+                int  _fd;
+            } poll_t _poll;
+        }
```
Comment (krytarowski) [inline src/core/sys/posix/signal.d]: For your interest, NetBSD 9.0 will ship with one new struct here.

Comment (ibuclaw) [reply src/core/sys/posix/signal.d]: Wonderful. I'm slowly coming round to thinking about that auto-generating these binding is likely the best way forward.

Comment (krytarowski) [reply src/core/sys/posix/signal.d]: On the other hand these things change rarely.

Comment (ibuclaw) [reply src/core/sys/posix/signal.d]: Yes, but currently you can't reliably support multiple versions of the same platform, and assuming the latest version is not a good way to go.

**Pull request merged**